interfaces/Portalicious/src/app/pages/project/project-registrations/project-registrations.page.ts

```diff
@@ -33,6 +34,8 @@ import {
 } from '~/services/paginate-query.service';
 import { ToastService } from '~/services/toast.service';
 
+import { RegistrationStatusEnum } from '../../../../../../../services/121-service/src/registration/enum/registration-status.enum';
```

Comment:
Linter should be complaining and/or fixing this to use from '@121-service` 🧐 

You might also wanna check your editor for this setting:

`"javascript.preferences.importModuleSpecifier": "non-relative",`